Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/ComponentConfig.purs

```diff
@@ -331,7 +331,7 @@ disabilityBannerConfig state =
       , titleColor = Color.purple
       , actionText = (getString UPDATE_PROFILE)
       , actionTextColor = Color.purple
-      , imageUrl = "ny_ic_accessibility_banner_img,https://assets.juspay.in/beckn/merchantcommon/images/ny_ic_accessibility_banner_img.png"
+      , imageUrl = "ny_ic_accessibility_banner_img,"<> (getAssetStoreLink FunctionCall) <>"ny_ic_accessibility_banner_img.png"
```

Comment:
@MercyQueen Can you check these image doesn't exist in S3

Reply:
https://assets.juspay.in/beckn/yatri/driver/images/ny_ic_accessibility_banner_img.png

Reply:
this image is at user side : https://assets.juspay.in/beckn/yatri/user/images/ny_ic_accessibility_banner_img.png